snap/snapcraft.yaml

```diff
@@ -133,6 +153,38 @@ parts:
     stage-packages:
       - inotify-tools
 
+  cuda-samples:
+    source: https://github.com/NVIDIA/cuda-samples
+    source-tag: v11.5
+    source-type: git
+    source-depth: 1
+    plugin: nil
+    build-packages:
+      - build-essential
+      - on amd64:
+        - libnvidia-compute-515-server
+        - nvidia-cuda-toolkit
+      - on arm64:
+        - libnvidia-compute-515-server
+        - nvidia-cuda-toolkit
```

Comment:
```suggestion
    build-packages:
      - on amd64,arm64:
        - build-essential
        - libnvidia-compute-515-server
        - nvidia-cuda-toolkit
```

Reply:
is that valid?

From https://snapcraft.io/docs/snapcraft-advanced-grammar

> The body of the on clause is taken into account if every (AND, not OR) selector is true for the build environment. Currently the only selectors supported are host architectures (e.g. amd64).

something cannot be simulntationlsy on amd64 AND on arm64, meaning none of those get installed?

Reply:
I should RTFM. Thanks.